src/Servers/IIS/IIS/perf/Microbenchmarks/IIS.Microbenchmarks.csproj

```diff
@@ -3,13 +3,14 @@
   <Import Project="..\..\..\build\testsite.props" />
 
   <PropertyGroup>
-    <!-- Using shorter assembly name instead of Microsoft.AspNetCore.Server.Kestrel.Performance because https://github.com/dotnet/BenchmarkDotNet/issues/498 -->
-    <AssemblyName>IIS.Performance</AssemblyName>
-    <RootNamespace>Microsoft.AspNetCore.Server.IIS.Performance</RootNamespace>
+    <!-- Using shorter assembly name instead of Microsoft.AspNetCore.Server.Kestrel.Microbenchmarks because https://github.com/dotnet/BenchmarkDotNet/issues/498 -->
+    <AssemblyName>IIS.Microbenchmarks</AssemblyName>
+    <RootNamespace>Microsoft.AspNetCore.Server.IIS.Microbenchmarks</RootNamespace>
     <TargetFramework>$(DefaultNetCoreTargetFramework)</TargetFramework>
     <OutputType>Exe</OutputType>
     <ServerGarbageCollection>true</ServerGarbageCollection>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <SkipMicrobenchmarksValidation>true</SkipMicrobenchmarksValidation>
```

Comment:
Should this be conditional on something❔

Reply:
No, IIS microbenchmarks don't work right now.